### PR TITLE
LPS-176161 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LPS-176161

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test

**Comment by melodywu** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+			CookieHelper._checkSelectedCookieValue(
+				cookieName = "GUEST_LANGUAGE_ID",
+				expectedCookieValue = "en_US");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-4. Previously enabled and set cookies are deleted when Site level configuration is Disabled."
+	@priority = 5
+	test VerifyFunctionalCookiesForNotFirstVisitInSiteLevelDisableImplicitMode {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifyFunctionalCookiesForNotFirstVisitInSiteLevelDisableImplicitMode";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase:116]: Please ass HttpOnly disabler for this test case

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+			CookieConsentPanel.verifyCookiesForCookieCategoriesAreNotPresent();
+
+			CookieHelper.checkSelectedCookieIsNotExist(cookieName = "GUEST_LANGUAGE_ID");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-10. Verify Site 1 cookies don't appear when visiting Site 2"
+	@priority = 5
+	test VerifySite1CookiesDontAppearWhenVisitingSite2 {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifySite1CookiesDontAppearWhenVisitingSite2";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase:388]: Please ass HttpOnly disabler for this test case #2

**Comment by melodywu** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase:118]: Done

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase (hunk 2 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase:391]: Done

**Comment by melodywu** [comment]
ci:test:bundle

**Pull request closed without merging**